notebooks/README.md

```diff
@@ -12,46 +12,37 @@ specific language governing permissions and limitations under the License.
 
 # 🤗 Optimum notebooks
 
-You can find here a list of the notebooks associated with each accelerator in 🤗 Optimum.                                                                                                                    |
+You can find here a list of the notebooks associated with each accelerator in 🤗 Optimum.
 
-## Optimum Habana examples
+## Optimum Habana
 
 | Notebook                                                                                                                                                                               | Description                                                                                                                                                                       |  Colab                                                                                                                                                                                                          |        Studio Lab                                                                                                                                                                                                   |
 |:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
 | [How to use DeepSpeed to train models with billions of parameters on Habana Gaudi](https://github.com/huggingface/optimum-habana/blob/main/notebooks/AI_HW_Summit_2022.ipynb) | Show how to use DeepSpeed to pre-train/fine-tune the 1.6B-parameter GPT2-XL for causal language modeling on Habana Gaudi. |  [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/optimum-habana/blob/main/notebooks/AI_HW_Summit_2022.ipynb) | [![Open in AWS Studio](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/huggingface/optimum-habana/blob/main/notebooks/AI_HW_Summit_2022.ipynb) |
 
-## Optimum Intel examples
+## Optimum Intel
+
+#### OpenVINO
```

Comment:
```suggestion
### OpenVINO
```
?

Reply:
Did this because it looks better this way in my opinion